.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+go:
+- 1.4.3
```

Comment:
Oh, I was using Go 1.5.1 to develop this, since it has the better-performing GC which probably benefits the proxy. It apparently does run in both versions though.